include/eve/arch/cpu/base.hpp

```diff
@@ -26,6 +28,8 @@ namespace eve::detail
     wide_storage() {}
     wide_storage(storage_type const &r) : data_(r) {}
 
+    wide_storage& operator=(storage_type const &r) { data_ = r; return *this; }
+
     EVE_FORCEINLINE storage_type const& storage() const & noexcept { return data_; }
     EVE_FORCEINLINE storage_type &      storage() &       noexcept { return data_; }
     EVE_FORCEINLINE storage_type        storage() &&      noexcept { return data_; }
```

Comment:
I don't suggest to change it at this time but I heard that it's a good idea to have a `const &&` as well for some weird magic.
Not sure why you bother with move at all, I don't think we have a single non trivially copiable type but anyways

Reply:
I wanted to be sure that calling `storage()` on temporary wide didn't fuck up.
I heard this `const&&` too, will look later